<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfiguration</class>
 <widget class="QDialog" name="DialogConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="myAccount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Copperplate Gothic Bold&quot;;</string>
   </property>
   <property name="title">
    <string>Data Configuration</string>
   </property>
   <widget class="QLabel" name="accountNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>New data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Copperplate Gothic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Dutch801 Rm BT&quot;;
color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>Old data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Dutch801 Rm BT&quot;;</string>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="old">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Copperplate Gothic Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Copperplate Gothic Bold&quot;;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>355</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="passW">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Copperplate Gothic Bold&quot;;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="emaiL">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Copperplate Gothic Bold&quot;;</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="phonenuM">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Copperplate Gothic Bold&quot;;</string>
       </property>
       <property name="text">
        <string>Phone Number</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogConfiguration</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
